R/expr-expr.R

```diff
@@ -3992,20 +3992,6 @@ expr__gather_every <- function(n, offset = 0) {
   })
 }
 
-# TODO-REWRITE: Requires $map_batches()
-# #' Print the value that this expression evaluates to and pass on the value
-# #'
-# #' @param fmt How to format the expression. Use `"{}"` where you want to print
-# #' the value of the expression.
-# #'
-# #' @inherit as_polars_expr return
-# #' @examples
-# #' df <- pl$DataFrame(foo = c(1, 1, 2))
-# #' df$select(pl$col("foo")$cum_sum()$inspect("value is: {}")$alias("bar"))
-# expr__format <- function(fmt) {
-#
-# }
```

Comment:
Since `$map_batches()` will likely not be included (cf #301) there's no need to keep this.

Reply:
Just to be clear, `<expr>$map_batches()` is included at this time. I merely question its usefulness.
And, it is still buggy (https://github.com/eitsupi/neo-r-polars/issues/151#issuecomment-2797139057).

Reply:
Right, I meant it like if we want to orient users to `purrr` / `mirai`, let's not use `map_batches()` too much internally.